Description: No intersphinx plugin for sphinx doc
 Do not use network when building.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-06-02

--- tempest-1.orig/doc/source/conf.py
+++ tempest-1/doc/source/conf.py
@@ -27,7 +27,6 @@ import os
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.todo',
               'sphinx.ext.viewcode',
               'oslosphinx'
